packages/app/src/App.tsx

```diff
@@ -108,3 +108,4 @@ const App = () => (
 );
 
 export default App;
+export const ICON_ANNOTATION = 'operate-first.cloud/logo-url';
```

Comment:
This is introducing circular dependency... let's not do that. If necessary create a separate file for constants...

Reply:
Resolved in the latest commit.